### PR TITLE
feat(posts): update uses

### PR DESCRIPTION
**Pull request by paramoNNNN**

### CODE REVIEW

File: content/blog/uses/index.md [hunk 1]
```diff
@@ -14,51 +14,48 @@ Let's start!
...
 - [Prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode): Format and beautify code.
-- [Project Manager](https://marketplace.visualstudio.com/items?itemName=alefragnani.project-manager): Manage projects (Save, open, delete, etc).
 - [TODO Highlight](https://marketplace.visualstudio.com/items?itemName=wayou.vscode-todo-highlight): Highlight todos in the code.
 - [Toggle Quotes](https://marketplace.visualstudio.com/items?itemName=BriteSnow.vscode-toggle-quotes): Toggle quotes. Very useful in JavaScript.
 - [vscode-styled-components](https://marketplace.visualstudio.com/items?itemName=jpoissonnier.vscode-styled-components): styled-components syntax highlighter.
 
 #### Desktop apps
 
-- [Adobe XD](https://www.adobe.com/products/xd.html) to design interfaces.
+- [Figma](https://www.adobe.com/products/xd.html) to design interfaces.
```
Comment (poeti8) [inline content/blog/uses/index.md:31]: Figma should be right before Firefox to maintain alphabetical order.

**Comment by poeti8** [comment]
Hmm something off with the package-lock.json file, maybe not commit it.

**Comment by paramoNNNN** [comment]
> Hmm something off with the package-lock.json file, maybe not commit it.

Reverted package-lock and it's still failing to build.

**Comment by paramoNNNN** [comment]
Seems relatable [Link](https://github.com/gatsbyjs/gatsby/issues/27607)

**Pull request merged**